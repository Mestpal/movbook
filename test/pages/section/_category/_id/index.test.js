import Vue from 'vue'
import Vuetify from 'vuetify'
import Vuex from 'vuex'
import { shallowMount, createLocalVue } from '@vue/test-utils'

import index from '@/pages/section/_category/_id/index'
Vue.use(Vuetify)
Vue.use(Vuex)
const localVue = createLocalVue()

describe('section category id index.vue page', () => {
  const $route = {
    params: {
      id: 11111111,
      category: 'Test'
    }
  }

  let wrapper
  beforeEach(() => {
    wrapper = shallowMount(index, {
      localVue,
      mocks: {
        $route
      },
      methods: {
        updateSelectedGenre () {
          return ''
        }
      }
    })
  })

  it('reders the page', () => {
    // wrapper.vm.updateSelectedGenre = jest.fn()
    // wrapper.vm.categoryTitle = jest.fn()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
