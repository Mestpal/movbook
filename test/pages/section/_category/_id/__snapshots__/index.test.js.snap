// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`section category id index.vue page reders the page 1`] = `
"<v-layout-stub tag=\\"div\\" row=\\"\\">
  <v-flex-stub tag=\\"div\\">
    <div>
      <h1>TEST</h1>
    </div>
    <contentsgrid-stub></contentsgrid-stub>
  </v-flex-stub>
</v-layout-stub>"
`;
